VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionPostsSearchFragment.java

```diff
@@ -35,6 +41,29 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
         return inflater.inflate(R.layout.fragment_discussion_search_posts, container, false);
     }
 
+    @Override
+    public void onViewCreated(View view, Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+
+        discussionTopicsSearchView.setQuery(searchQuery, false);
+        discussionTopicsSearchView.setOnQueryTextListener(new SearchView.OnQueryTextListener() {
+            @Override
+            public boolean onQueryTextSubmit(String query) {
+                if (query == null || query.trim().length() == 0)
```

Comment:
This check can be replaced with `StringUtils.isBlank()` from the Apache Commons library, which we use. Note that it checks for `null` as well.